.chloggen/operator-template-fix.yaml

```diff
@@ -0,0 +1,12 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, networkExplorer, operator, chart, other)
+component: operator
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fix helm template bug where some non-default Operator auto-instrumentation values can cause deployment failures
+# One or more tracking issues related to the change
+issues: [938,943]
```

Comment:
I think we should not link both issue and PR for the issue

Reply:
Going to only link the PR for now.

Reply:
In the collector, we link issue and use PR as a fallback if there is no issue for the change

Reply:
Done